jib-core/src/main/java/com/google/cloud/tools/jib/blob/Blob.java

```diff
@@ -30,4 +35,28 @@
    * @throws IOException if writing the BLOB fails
    */
   BlobDescriptor writeTo(OutputStream outputStream) throws IOException;
+
+  /**
+   * Writes the BLOB to a file, optionally with an exclusive lock.
+   *
+   * @param jsonFile the file to write to
+   * @param acquireLock if true then acquire an exclusive lock
+   * @return the {@link BlobDescriptor} of the written BLOB
+   * @throws IOException if writing the BLOB fails
+   */
+  default BlobDescriptor writeTo(Path jsonFile, boolean acquireLock) throws IOException {
```

Comment:
Oh I was referring to placing the method in `Blobs#writeToFile` (similar to `Blobs#writeToString` and `Blobs#writeToByteArray` - `Blobs` is intended to be static methods that operate on `Blob`). This will help keep the `Blob` interface simple and avoid having to use default interface methods.

Reply:
Ah, I didn't think of Blobs. Will look.